regression/pages/studio/unit_page.py

```diff
@@ -0,0 +1,20 @@
+"""
+Unit Page for Studio
+"""
+
+from edxapp_acceptance.pages.studio.container import ContainerPage
+
+
+class UnitPageExtended(ContainerPage):
+    """
+    Unit Page for Studio
+    """
+    def view_live_version(self):
+        """
+        Clicks View live version button on Units page
+        """
+        self.q(
+            css='.action-button'
+            '[title="Open the courseware in the LMS"]'
+        ).click()
+        self.browser.switch_to_window(self.browser.window_handles[-1])
```

Comment:
I'm not sure why this is important (I'm sure it is though). Could you give some details in the docstring? (Something like "click this button, which opens up a new window".)